scripts/py_matter_yamltests/matter_yamltests/yaml_loader.py

```diff
@@ -233,9 +234,11 @@ def __rule_node_id_and_group_id_are_mutually_exclusive(self, content):
 
     def __rule_group_step_should_not_expect_a_response(self, content):
         if 'groupId' in content and 'response' in content:
-            response = content.get('response')
-            if 'value' in response or 'values' in response:
-                raise TestStepGroupResponseError(content)
+            raise TestStepGroupResponseError(content)
+
+    def __rule_group_step_should_not_target_an_endpoint(self, content):
+        if 'groupId' in content and 'endpoint' in content:
```

Comment:
Can we also check for `'response'`?  I would think there should be no `'response'` on group commands.